x/concentrated-liquidity/bench_test.go

```diff
@@ -267,7 +283,9 @@ func BenchmarkGetTickLiquidityNetInDirection(b *testing.B) {
 }
 
 func BenchmarkGetTickLiquidityForFullRange(b *testing.B) {
+	b.Helper()
 	runBenchmark(b, func(b *testing.B, s *BenchTestSuite, pool types.ConcentratedPoolExtension, largeSwapInCoin sdk.Coin, currentTick int64) {
+		b.Helper()
```

Comment:
Same question here for the Helper method, what is the significance of having this here?

Reply:
So, it is a marker, and it also allows better concurrency (same link as above)

https://cs.opensource.google/go/go/+/go1.21.0:src/testing/testing.go;l=1127